cms/templates/library.html

```diff
@@ -29,8 +33,8 @@
             {
                 isUnitPage: false,
                 page_size: 10,
-                canEdit: ${"true" if can_edit else "false"},
-                showChildrenPreviews: ${'true' if show_children_previews else 'false'}
+                canEdit: ${can_edit | n, dump_js_escaped_json},
+                showChildrenPreviews: ${context_library.show_children_previews | n, dump_js_escaped_json}
```

Comment:
@adampalay: One thing to understand is that None => `null`, it won't go to `true` or `false`.  If you aren't sure if you have a boolean, you can always do:
```
canEdit: ${bool(can_edit) | n, dump_js_escaped_json},
```
This is just an FYI in case you aren't sure what you have.